synapse/models/orgs.py

```diff
@@ -10,6 +10,12 @@
     'partnership',
 )
 
+reqlevels = (
+    (10, 'may'),
+    (20, 'should'),
+    (30, 'must'),
```

Comment:
Literature (and practice) in requirements engineering tends to use SHALL more than MUST. I dislike MAY as it implies permission, which sounds odd for a requirement.

Reply:
Yea, i've def seen `shall` more commonly, i'll adjust. Is there another word you would use to convey that a given requirement might be "preferred" but is not as strong as `should` ? I've def seen `may` used ( and felt icky about it at the time for the same reason :) )